<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Capture</class>
 <widget class="QWidget" name="Capture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>501</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGroupBox" name="setupGroup_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>CCD</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel1_6">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Device:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="CCDCaptureCombo"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textLabel1_7">
            <property name="toolTip">
             <string>Prefix to append to the beginning of file names</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Prefix:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KLineEdit" name="prefixIN"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="exposureLabel_2">
            <property name="toolTip">
             <string>Set the exposure time in seconds for individual images, if applicable</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Exposure:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KDoubleNumInput" name="exposureIN">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="textLabel1_4">
            <property name="toolTip">
             <string>Number of images to capture</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Count:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="KIntNumInput" name="countIN">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1024</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="textLabel1_8">
            <property name="toolTip">
             <string>Delay in seconds between consecutive images</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Delay:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="KIntNumInput" name="delayIN"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_8">
            <property name="toolTip">
             <string>Apply filter to image after capture to enhance it</string>
            </property>
            <property name="text">
             <string>Filters:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="filterCombo">
            <item>
             <property name="text">
              <string>--</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="ISOCheck">
          <property name="toolTip">
           <string>Add time stamp to the file name</string>
          </property>
          <property name="text">
           <string>Add ISO 8601 time stamp</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="displayCheck">
          <property name="text">
           <string>Display in FITS Viewer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="darkSubCheck">
          <property name="text">
           <string>Auto dark subtract</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="setupGroup">
       <property name="title">
        <string>Filter</string>
       </property>
       <layout class="QGridLayout" name="_10">
        <property name="margin">
         <number>10</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0" rowspan="2">
         <layout class="QVBoxLayout" name="_11">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_5_3">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Device:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1_2_3">
            <property name="toolTip">
             <string>Number of images to capture</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="KComboBox" name="FilterPosCombo"/>
        </item>
        <item row="0" column="1">
         <widget class="KComboBox" name="FilterCaptureCombo"/>
        </item>
        <item row="2" column="0" colspan="2">
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>21</width>
            <height>81</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>267</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_7">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="startB">
         <property name="text">
          <string>&amp;Start</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>S&amp;top</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="progressGroup">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="progressLabel_2">
          <property name="text">
           <string>Progress:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentImgCountOUT">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_4_2_3">
          <property name="text">
           <string>of</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fullImgCountOUT">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="completedLabel_2">
          <property name="text">
           <string>completed</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>51</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="imgProgress">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
